test/decl/var/result_builders.swift

```diff
@@ -18,11 +18,10 @@ typealias typename = Inventor
 @Maker // expected-error {{result builder attribute 'Maker' can only be applied to a variable if it defines a getter}}
 var global: Int
 
-// FIXME: should this be allowed?
 @Maker
 var globalWithEmptyImplicitGetter: Int {}
-// expected-error@-1 {{missing return in accessor expected to return 'Int'}}
-// expected-error@-3 {{result builder attribute 'Maker' can only be applied to a variable if it defines a getter}}
+// expected-error@-1{{result builder 'Maker' does not implement any 'buildBlock' or a combination of 'buildPartialBlock(first:)' and 'buildPartialBlock(accumulated:next:)' with sufficient availability for this call site}}
+// missing return expectations moved to `SILOptimizer/missing_returns`
```

Comment:
Nit: There's no missing return expectation for this case :)

Reply:
sorry, i don't think i follow... i added similar code to the `missing_returns` tests but this formulation is now valid if the result builder protocol is satisfied. there's no expectation b/c this is now 'fixed'. seemed like it should be moved there though since this test doesn't do the SIL pass. should this be worded differently, or am i misunderstanding?

Reply:
Sorry yeah I just meant this should be worded differently since the issue isn't a lack of return